crates/tarball/Cargo.toml

```diff
@@ -29,7 +29,7 @@ tar          = { workspace = true }
 tokio        = { workspace = true }
 zune-inflate = { workspace = true }
 tracing      = { workspace = true }
-rayon.workspace = true
+rayon = { workspace = true }
```

Comment:
You didn't align the column. You can either do it manually or run `taplo format`.